assets/xml/objects/object_rb.xml

```diff
@@ -1,44 +1,44 @@
-        <DList Name="object_rb_DL_000B80" Offset="0xB80" />
-        <DList Name="object_rb_DL_000C18" Offset="0xC18" />
-        <DList Name="object_rb_DL_000C28" Offset="0xC28" />
-        <Texture Name="object_rb_Tex_000CD8" OutName="tex_000CD8" Format="rgba16" Width="8" Height="16" Offset="0xCD8" />
-        <Texture Name="object_rb_Tex_000DD8" OutName="tex_000DD8" Format="rgba16" Width="32" Height="32" Offset="0xDD8" />
-        <Texture Name="object_rb_Tex_0015D8" OutName="tex_0015D8" Format="rgba16" Width="32" Height="32" Offset="0x15D8" />
-        <Limb Name="object_rb_Standardlimb_001DD8" Type="Standard" EnumName="OBJECT_RB_LIMB_01" Offset="0x1DD8" />
-        <Limb Name="object_rb_Standardlimb_001DE4" Type="Standard" EnumName="OBJECT_RB_LIMB_02" Offset="0x1DE4" />
-        <Limb Name="object_rb_Standardlimb_001DF0" Type="Standard" EnumName="OBJECT_RB_LIMB_03" Offset="0x1DF0" />
-        <Limb Name="object_rb_Standardlimb_001DFC" Type="Standard" EnumName="OBJECT_RB_LIMB_04" Offset="0x1DFC" />
-        <Limb Name="object_rb_Standardlimb_001E08" Type="Standard" EnumName="OBJECT_RB_LIMB_05" Offset="0x1E08" />
-        <Limb Name="object_rb_Standardlimb_001E14" Type="Standard" EnumName="OBJECT_RB_LIMB_06" Offset="0x1E14" />
-        <Limb Name="object_rb_Standardlimb_001E20" Type="Standard" EnumName="OBJECT_RB_LIMB_07" Offset="0x1E20" />
-        <Limb Name="object_rb_Standardlimb_001E2C" Type="Standard" EnumName="OBJECT_RB_LIMB_08" Offset="0x1E2C" />
-        <Limb Name="object_rb_Standardlimb_001E38" Type="Standard" EnumName="OBJECT_RB_LIMB_09" Offset="0x1E38" />
-        <Limb Name="object_rb_Standardlimb_001E44" Type="Standard" EnumName="OBJECT_RB_LIMB_0A" Offset="0x1E44" />
-        <Limb Name="object_rb_Standardlimb_001E50" Type="Standard" EnumName="OBJECT_RB_LIMB_0B" Offset="0x1E50" />
-        <Limb Name="object_rb_Standardlimb_001E5C" Type="Standard" EnumName="OBJECT_RB_LIMB_0C" Offset="0x1E5C" />
-        <Limb Name="object_rb_Standardlimb_001E68" Type="Standard" EnumName="OBJECT_RB_LIMB_0D" Offset="0x1E68" />
-        <Limb Name="object_rb_Standardlimb_001E74" Type="Standard" EnumName="OBJECT_RB_LIMB_0E" Offset="0x1E74" />
-        <Limb Name="object_rb_Standardlimb_001E80" Type="Standard" EnumName="OBJECT_RB_LIMB_0F" Offset="0x1E80" />
-        <Limb Name="object_rb_Standardlimb_001E8C" Type="Standard" EnumName="OBJECT_RB_LIMB_10" Offset="0x1E8C" />
-        <Limb Name="object_rb_Standardlimb_001E98" Type="Standard" EnumName="OBJECT_RB_LIMB_11" Offset="0x1E98" />
-        <Skeleton Name="object_rb_Skel_001EE8" Type="Normal" LimbType="Standard" LimbNone="OBJECT_RB_LIMB_NONE" LimbMax="OBJECT_RB_LIMB_MAX" EnumName="object_rb_Limbs" Offset="0x1EE8" />
+        <Animation Name="gLeeverSpinAnim" Offset="0x1E4" />
+        <DList Name="gLeeverBottomHalfDlist" Offset="0x200" />
+        <DList Name="gLeeverBottomHalfWrapperDlist2" Offset="0x2D0" />
+        <DList Name="gLeeverBottomHalfWrapperDlist1" Offset="0x2E0" />
```

Comment:
We use DL as a suffix for dlists, and we try to avoid putting anything after that prefix when those are properly named
```suggestion
        <DList Name="gLeeverBottomHalfDL" Offset="0x200" />
        <DList Name="gLeeverBottomHalfWrapper2DL" Offset="0x2D0" />
        <DList Name="gLeeverBottomHalfWrapper1DL" Offset="0x2E0" />
```